openmetadata-service/src/main/java/org/openmetadata/service/resources/automations/WorkflowMapper.java

```diff
@@ -2,30 +2,19 @@
 
 import org.openmetadata.schema.entity.automations.CreateWorkflow;
 import org.openmetadata.schema.entity.automations.Workflow;
-import org.openmetadata.schema.services.connections.metadata.OpenMetadataConnection;
-import org.openmetadata.service.OpenMetadataApplicationConfig;
 import org.openmetadata.service.mapper.EntityMapper;
-import org.openmetadata.service.util.OpenMetadataConnectionBuilder;
 
 public class WorkflowMapper implements EntityMapper<Workflow, CreateWorkflow> {
-  private final OpenMetadataApplicationConfig openMetadataApplicationConfig;
-
-  public WorkflowMapper(OpenMetadataApplicationConfig config) {
-    this.openMetadataApplicationConfig = config;
-  }
 
   @Override
   public Workflow createToEntity(CreateWorkflow create, String user) {
-    OpenMetadataConnection openMetadataServerConnection =
-        new OpenMetadataConnectionBuilder(openMetadataApplicationConfig).build();
```

Comment:
we don't need `openMetadataApplicationConfig` stored in the workflow. It gets added at runtime by the `pipleineServiceClient`.